doc/sphinx/proof-engine/tactics.rst

```diff
@@ -2227,16 +2227,19 @@ and an explanation of the underlying technique.
       then :n:`injection @ident` first introduces the hypothesis in the local
       context using :n:`intros until @ident`.
 
-   .. exn:: Not a projectable equality but a discriminable one.
-      :undocumented:
-
    .. exn:: Nothing to do, it is an equality between convertible terms.
       :undocumented:
 
    .. exn:: Not a primitive equality.
       :undocumented:
 
    .. exn:: Nothing to inject.
+      This error is given when one side of the equality is not a constructor.
```

Comment:
It's possible that you need to skip a line before the description (and that would explain the failing build).

Reply:
ok, I tried.